### ISSUE TITLE
"Базова українська + російська" розкладка не працює під Windows 11

### ISSUE THREAD

**Issue by OleksandrMatiash** [neochief/ukrainian-typographic-layouts#6]
При спробі перейти на дану розкладку мовна панель перезагружається і вибирається попередня активна розкладка.

**Comment by neochief** [comment]
Вітаю! Так, проблема відома, але поки що для неї немає рішення.

**Comment by ALERTua** [comment]
це нетипово навіть для сирої Windows 11 Insider. можемо спробувати розбиратись:
- видаліть всі розкладки, залиште тільки базову англійську US English
- зайдть в адміністративні мовні налаштування за допомогою
`"C:\Windows\system32\rundll32.exe" shell32.dll,Control_RunDLL intl.cpl,,1`
- виберіть опцію копіювання налаштувань для всіх юзерів
- поставте знизу обидві галочки
- підтвердіть копіювання
- перезавантажте OS
на цьому етапі у вас повинно бути на всіх юзерах все чітко: одна штатна розкладка US English

- тепер поступово одну за однією (встановіть і) додайте ваші кастомні розкладки і після кожної копіюйте налаштування для всіх юзерів через адміністративні мовні налаштування.
- видаліть US English, змініть порядок розкладок на бажаний
- встановіть хоткеї для кожної розкладки, якщо хочете
- скопіюйте налаштування через адміністративні мовні налаштування для всіх юзерів
- перезавантажте OS
- перевірте, чи все працює

**Comment by neochief** [comment]
@ALERTua А у вас розкладки "завелись" на Win11? Я їх тестував там десь рік назад і у мене все закінчилось фіаско. Самі розкладки не встановлювалися, а тулза для редагування розкладок на той час не була готова для Win11.

**Comment by ALERTua** [comment]
@neochief так, я постійно юзаю англійську Бірмана, та `tukrrus` на Windows 11 Insider, на даний момент — 22635.3212. Протягом перехідного періода на українську, юзав також `trusukr`. також, в мене встановлені хоткеї alt+shift+1, alt+shift+2 (і раніше — alt+shift+3) для прямого переключення розкладками. це все коректно працює тільки після копіювання на всіх юзерів. інакше вінда дуже сильно намагається всунути свою US English першою розкладкою і наламує все. саме копіювання налаштувань на всіх юзерів і є працюючим воркераундом вже багато років.
Також все працює на робочих машинах win10 19045 та win11 22631.3155

**Comment by neochief** [comment]
@ALERTua Дякую, мабуть тоді час мені спробувати все ще раз.
